src/aat/resources/features/F-1021 - On Behalf Of User Id after Case created Get events by case ID/S-1021.1.td.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "must return list of audit events successfully for a case",
+  "_guid_": "S-1021.1",
+  "_extends_": "F-1021_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "uses the case-reference of the case just created",
+    "contains a token created as in S-597-Prerequisite",
```

Comment:
Are these two entries needed?  Not able to find references to them in the other files.

Reply:
"contains a token created as in S-597-Prerequisite", - this is not needed, other is needed